lib/bokken/curriculum/skill.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Bokken.Curriculum.Skill do
+  @moduledoc """
+  A skill a mentor has / a ninja is learning
+  """
+  use Bokken.Schema
+
+  @required_fields [:name, :description]
+
+  schema "skills" do
+    field :name, :string
```

Comment:
Shouldn't this be a `citext`?

Reply:
It is specified as citext in the migration file. As far as the model is concerned, it is a string though